projects/packages/forms/src/blocks/contact-form/components/jetpack-field-consent.js

```diff
@@ -45,31 +58,54 @@ const JetpackFieldConsent = ( {
 				) }
 				insertBlocksAfter={ insertBlocksAfter }
 			/>
+			<JetpackFieldDimensionControls
+				clientId={ clientId }
+				setAttributes={ setAttributes }
+				width={ width }
+			/>
+			<InspectorControls group="styles">
+				<ToolsPanel
+					panelId={ clientId }
+					label={ __( 'Color', 'jetpack-forms' ) }
+					resetAll={ () => setAttributes( { labelColor: undefined } ) }
+					dropdownMenuProps={ toolsPanelDropdownMenuProps }
+				>
+					<div className="jetpack-field-controls__color-settings">
+						<ColorGradientSettingsDropdown
+							__experimentalIsRenderedInSidebar
+							panelId={ clientId }
+							gradients={ [] }
+							disableCustomGradients
+							settings={ [
+								{
+									colorValue: attributes.labelColor,
+									onColorChange: value => setAttributes( { labelColor: value } ),
+									label: __( 'Label Text', 'jetpack-forms' ),
+									clearable: true,
+								},
+							] }
+						/>
+					</div>
+				</ToolsPanel>
+				<ToolsPanel>
+					<div style={ { gridColumn: '1 / -1' } }>
```

Comment:
Perhaps this could be a shared class as it will be reused a bit by the looks of this PR.